cmake/sca/eclair/Kconfig.eclair

```diff
@@ -0,0 +1,165 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2023, BUGSENG Srl
+
+mainmenu "ECLAIR Configurations"
+
+menu "ECLAIR env configurations"
+
+choice ECLAIR_RULESET
+	prompt "ECLAIR analysis ruleset"
+	default ECLAIR_RULESET_FIRST_ANALYSIS
+
+config ECLAIR_RULESET_FIRST_ANALYSIS
+	bool "first analysis [EXPERIMENTAL]"
+	select EXPERIMENTAL
```

Comment:
Instead of selecting `EXPERIMENTAL` everywhere, you could add a hidden default selected option:

```
config ECLAIR
	bool
	default y
	select EXPERIMENTAL
```

Reply:
I like the idea of having a default selected option for everything. But in this case wouldn't it be better to clear the experimental when a feature was tested enough?